src/packet_analysis/protocol/tcp/TCPSessionAdapter.cc

```diff
@@ -1670,29 +1670,66 @@ int TCPSessionAdapter::ParseTCPOptions(const struct tcphdr* tcp, bool is_orig)
 
 		unsigned int opt_len;
 
+		// Non-standard TCP option kinds should not be common
+		// https://www.iana[.]org/assignments/tcp-parameters/tcp-parameters.xhtml
```

Comment:
Minor nit, but I don't understand why this URL is sanitized. Just makes it harder to actually reference.

Reply:
It's a force of habit, I guess. I'll remove it here.